Makefile

```diff
@@ -127,7 +127,7 @@ gas: $(GAS)
 # convenience targets
 all: components tethers isos vic-machine vic-ui
 tools: $(GOIMPORTS) $(GVT) $(GOLINT) $(SWAGGER) $(GAS) goversion
-check: goversion goimports govet golint copyright whitespace gas
+check: goversion goimports govet lintcode copyright whitespace gas
```

Comment:
More seriously, we will likely never use this repo for more than `go` (and possibly some `javascript`).  And in the case where we need to call more specific linters, we'll add targets specifically for those linters.